docs/node-server-sdk.iclientconfig.eventtracking.md

```diff
@@ -1,18 +1,18 @@
 <!-- Do not edit this file. It is automatically generated by API Documenter. -->
 
-[Home](./index.md) &gt; [@eppo/node-server-sdk](./node-server-sdk.md) &gt; [IClientConfig](./node-server-sdk.iclientconfig.md) &gt; [eventIngestionConfig](./node-server-sdk.iclientconfig.eventingestionconfig.md)
+[Home](./index.md) &gt; [@eppo/node-server-sdk](./node-server-sdk.md) &gt; [IClientConfig](./node-server-sdk.iclientconfig.md) &gt; [eventTracking](./node-server-sdk.iclientconfig.eventtracking.md)
 
-## IClientConfig.eventIngestionConfig property
+## IClientConfig.eventTracking property
 
 Configuration settings for the event dispatcher
 
 **Signature:**
 
 ```typescript
-eventIngestionConfig?: {
+eventTracking?: {
         batchSize?: number;
         deliveryIntervalMs?: number;
-        disabled?: boolean;
+        enabled?: boolean;

```

Comment:
Nit: Do we actually need this at all? The eventTracking field is optional. If it's missing, it's disabled. If it's present, it's enabled. Forcing users to set this seems like an extra step that could be missed leading to confusion.

Reply:
cool, that makes sense.